src/recenter/module.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @module ngeo recenter namespace
+ */
+goog.provide('ngeo.recenter.module');
```

Comment:
This module is not necessary (there is no recenter top level module). You can recycle it by renaming it `ngeo.map.module` and moving it there if it does not already exist.